app/actions/StatisticsActions.js

```diff
@@ -633,18 +629,23 @@ export const dailyBalancesStats = (opts) => {
   let balance = { spendable: 0, locked: 0, total: 0, sent: 0, received: 0,
     voted: 0, revoked: 0, ticket: 0 };
 
-  const aggStartFunction = (opts) => {
-    opts.series = [ ...opts.series,
-      { name: "sent", type: VALUE_TYPE_ATOMAMOUNT },
-      { name: "received", type: VALUE_TYPE_ATOMAMOUNT },
-      { name: "voted", type: VALUE_TYPE_ATOMAMOUNT },
-      { name: "revoked", type: VALUE_TYPE_ATOMAMOUNT },
-      { name: "ticket", type: VALUE_TYPE_ATOMAMOUNT },
-      { name: "stakeRewards", type: VALUE_TYPE_ATOMAMOUNT },
-      { name: "stakeFees", type: VALUE_TYPE_ATOMAMOUNT },
-      { name: "totalStake", type: VALUE_TYPE_ATOMAMOUNT },
-    ];
-    startFunction(opts);
+  const aggStartFunction = () => {
+    startFunction({
+      series : [
```

Comment:
You removed the [ ...opt.series ] so now it's missing the immature and immatureNonWallet columns